czi-to-ome-tiff/xslt/instrument.xsl

```diff
@@ -1,38 +1,41 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-#
-# Comments in this template will generally be pointers from spec to spec
-# Example:
-#   Get Instrument Info
-#   zisraw/Instrument.xsd: 45
-#   ome/ome.xsd: 979
-#
-# This means that for more details on how this section of the template was created
-# view line 45 of the zisraw/Instrument.xsd file and view line 979 of the ome/ome.xsd file.
-#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--->
+<!-- #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ # # Comments in this template will generally be pointers from spec to spec # Example: # Get Instrument Info # zisraw/Instrument.xsd: 45 #
```

Comment:
something looks wrong with the newlines here...?

Reply:
🤷‍♀ Idk